### PR TITLE
Updated Base address and completing party + changed port number

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#17118

*Description of changes:*
- Updated Base Address and Completing Party components + respective import lines
- Changed the port number so that it's the same as before
- This ticket is waiting on https://github.com/bcgov/bcrs-shared-components/pull/194 to be merged

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ export default defineConfig(() => {
       },
       extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json', '.vue']
     },
+    server: {
+      host: true,
+      port: 8080
```
Comment (JazzarKarim) [inline vite.config.js:44]: Before:
before

After:
after

Now, the links will work the exact same as before to not confuse other Devs. For example: http://localhost:8080/incorporation-people-roles?id=TAEUG9DPGd

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-create-dev--pr-560-k4z7rkeh.web.app

File: vite.config.js (hunk 1 above)
Comment (seeker25) [reply vite.config.js:44]: stealing this

Comment (JazzarKarim) [reply vite.config.js:44]: All yours Travis 😎

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-create-dev--pr-560-k4z7rkeh.web.app

File: vite.config.js (hunk 1 above)
Comment (severinbeauvais) [reply vite.config.js:44]: 👍 👍 👍

File: package.json [hunk 2]
```diff
@@ -92,7 +93,7 @@
     "vite-plugin-environment": "^1.1.3",
     "vitest": "0.33.0",
     "vue-class-component": "7.2.6",
-    "vue-plugin-helper-decorator": "^0.0.12",
+    "vue-plugin-helper-decorator": "0.0.11",
```
Comment (severinbeauvais) [inline package.json:96]: What issue does this pre-empt/solve?

Comment (JazzarKarim) [reply package.json:96]: I thought it solved the above issue, but it didn't 😟

It was fixed at first but after shutting down the server and running again, I got the exact same issue. I'll probably revert this back.

**Comment by severinbeauvais** [comment]
Ah, can't fully test BaseAddress (in temp URL) because AddressComplete key is missing.

cc: @pwei1018 How do we fix this?

**Comment by severinbeauvais** [comment]
> I'm thinking of changing BaseAddress to fix this issue.

The shared BaseAddress? Go for it. Let me know how I can help.

**Comment by JazzarKarim** [comment]
> > I'm thinking of changing BaseAddress to fix this issue.
> 
> The shared BaseAddress? Go for it. Let me know how I can help.

Yes! I've tried everything in order to try and fix this without touching the shared component but to no avail. So, I'm onto plan B now.

File: package.json (hunk 2 above)
Comment (severinbeauvais) [reply package.json:96]: I feel for ya. Good job looking into it.

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-create-dev--pr-560-k4z7rkeh.web.app

**Comment by JazzarKarim** [comment]
> I can't seem to fix this error no matter what I try: source map error
> 
> Nothing out there also helped fixed my problem. It all has to do with the vue-plugin-helper-decorator package that's being used in BaseAddress. The odd thing is that the project ran normally at first. It also runs normally in the temporary URL. It's just spinning up the local environment that doesn't work anymore suddenly.
> 
> I'm thinking of changing BaseAddress to fix this issue.

I've finally fixed this error 😁

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-create-dev--pr-560-k4z7rkeh.web.app

File: package.json (hunk 2 above)
Comment (severinbeauvais) [reply package.json]: Obsolete package. Now using "vue-plugin-helper-decorator". Resolving.

**Comment by severinbeauvais** [comment]
I've testing AddressComplete and PDF Upload at the temporary URL (Coop IA) -- looks great!

**Comment by JazzarKarim** [comment]
> I've testing AddressComplete and PDF Upload at the temporary URL (Coop IA) -- looks great!

Oh, AddressComplete is now working in the Temp URL! That's pretty cool 🙌

**Review by severinbeauvais — APPROVED** [review]
Would be nice to get other reviews, but if you're dying to merge this, go for it.

**Review by ketaki-deodhar — APPROVED** [review]

**Pull request merged**